modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/RelatedObjectEntryResourceImpl.java

```diff
@@ -54,6 +57,44 @@
 public class RelatedObjectEntryResourceImpl
 	extends BaseRelatedObjectEntryResourceImpl {
 
+	@Override
+	public void deleteObjectRelationshipMappingTableValues(
+			String previousPath, Long objectEntryId,
+			String objectRelationshipName, Long relatedObjectEntryId)
+		throws Exception {
+
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153324"))) {
+			throw new NotFoundException();
+		}
+
+		ObjectDefinition systemObjectDefinition = _getSystemObjectDefinition(
+			previousPath);
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.
+				getObjectRelationshipByObjectDefinitionId(
+					systemObjectDefinition.getObjectDefinitionId(),
+					objectRelationshipName);
+
+		_getRelatedObjectEntry(
```

Comment:
Why is this method call needed?

According to the method name, this should return an ObjectEntry but is not using that ObjectEntry that is producing. Unless the method does something more, I think this call is unnecessary.

Reply:
I think it is needed to check if the object exists before trying to perform any relationship operation

Reply:
I curse the day that Brian forced us to send a 404 in the DELETE endpoints

Reply:
Then... should we check for both? I mean, we are checking if the Subject exists, aren't we?
Should we check if the user exists?

Reply:
mmm, good point. We could review both validations on monday